src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,8 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
```

Comment:
The 'movies' prop is used without propType validation. Make sure to add propTypes to validate the 'movies' prop.